### PR TITLE
Fix Multiple Notes on SSH Keys page

### PR DESCRIPTION
**Pull request by laurapacilio**
### What
Recently a contributor added a note to this page about SSH keys. The note is right next to another note in the text, which is not an ideal user experience. I'm making some fixes to make this page align with our style guide and be generally easier to read :) 

### Why
Multiple notes in a row are confusing and often distracting for users. I also made some style adjustments (bolding UI elements, removing contractions, etc.), updated the UI button to match the current UI (it's SSH Keys, not Manage SSH Keys), and broke the section up so it's easier for users to find the action that they want to do. 

### Screenshots
Screen Shot 2022-11-16 at 7 18 33 PM

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx [hunk 1]
```diff
@@ -19,26 +19,30 @@ To assign a key to a workspace, go to its settings and choose a previously added
...
+1. Paste the text of the private key in the **Private SSH Key** field.
+1. Click **Add Private SSH Key**.
+
+The new key appears in the list of keys on the page. 
 
-   The exact command to create a PEM formatted SSH keypair depends on your OS, but is usually something like `ssh-keygen -t rsa -m PEM -f "/Users/<NAME>/.ssh/service_terraform" -C "service_terraform_enterprise"`. This creates a `service_terraform` file with the private key, and a `service_terraform.pub` file with the public key.
-1. Enter a name for the key in the "Name" field. Choose something identifiable, since the name is the only way to tell two SSH keys apart once the key text is hidden.
-1. Paste the text of the **private key** in the "Private SSH Key" field.
-1. Click the "Add Private SSH Key" button.
+If you upload an invalid ssh key, you should upload the correct ssh key and push a new commit for the new key to take effect.
```
Comment (laurapacilio) [inline website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx:36]:
```suggestion
If you upload an invalid SSH key, upload the correct key and push a new commit for the new key to take effect.
```

File: website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx [hunk 2]
```diff
@@ -19,26 +19,30 @@ To assign a key to a workspace, go to its settings and choose a previously added
...
+1. Go to your organization's settings and choose **SSH Keys**. This page has a form for adding new keys and a list of existing keys.
+1. Obtain a PEM formatted SSH keypair that Terraform Cloud can use to download modules during a Terraform run. You might already have an appropriate key; if not, create one on a secure workstation and distribute the public key to your VCS provider(s). Do not use or generate a key that has a passphrase. Git is running non-interactively and cannot prompt for it.
+
+   The exact command to create a PEM formatted SSH keypair depends on your operating system, but is usually something like `ssh-keygen -t rsa -m PEM -f "/Users/<NAME>/.ssh/service_terraform" -C "service_terraform_enterprise"`. This creates a `service_terraform` file with the private key, and a `service_terraform.pub` file with the public key.
```
Comment (laurapacilio) [inline website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx:29]:
```suggestion
   The exact command to create a PEM formatted SSH keypair depends on your operating system. The following example command creates a `service_terraform` file with the private key, and a `service_terraform.pub` file with the public key.
   ```bash
   ssh-keygen -t rsa -m PEM -f "/Users/<NAME>/.ssh/service_terraform" -C "service_terraform_enterprise"
   ```
```

File: website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx [hunk 3]
```diff
@@ -19,26 +19,32 @@ To assign a key to a workspace, go to its settings and choose a previously added
...
+1. Go to your organization's settings and choose **SSH Keys**. This page has a form for adding new keys and a list of existing keys.
+1. Obtain a PEM formatted SSH keypair that Terraform Cloud can use to download modules during a Terraform run. You might already have an appropriate key; if not, create one on a secure workstation and distribute the public key to your VCS provider(s). Do not use or generate a key that has a passphrase. Git is running non-interactively and cannot prompt for it.
+
+   The exact command to create a PEM formatted SSH keypair depends on your operating system. The following example command creates a `service_terraform` file with the private key, and a `service_terraform.pub` file with the public key.
+   ```bash
+   ssh-keygen -t rsa -m PEM -f "/Users/<NAME>/.ssh/service_terraform" -C "service_terraform_enterprise"
```
Comment (laurapacilio) [inline website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx:31]:
```suggestion
   The exact command to create a PEM formatted SSH keypair depends on your operating system. The following example command creates a `service_terraform` file with the private key, and a `service_terraform.pub` file with the public key.
   ```bash
   ssh-keygen -t rsa -m PEM -f "/Users/<NAME>/.ssh/service_terraform" -C "service_terraform_enterprise"
   ```
```

File: website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx [hunk 4]
```diff
@@ -19,26 +19,32 @@ To assign a key to a workspace, go to its settings and choose a previously added
...
-## Adding and Deleting Keys
-
-To add or delete an SSH private key, use the main menu to go to your organization's settings and choose "Manage SSH Keys" from the navigation sidebar. This page has a form for adding new keys and a list of existing keys.
+## Adding Keys
 
 To add a key:
 
-1. Obtain a PEM formatted SSH keypair that Terraform Cloud can use to download modules during a Terraform run. You might already have an appropriate key; if not, create one on a secure workstation and distribute the public key to your VCS provider(s). Do not use or generate a key that has a passphrase; Git is running non-interactively and won't be able to prompt for it.
+1. Go to your organization's settings and choose **SSH Keys**. This page has a form for adding new keys and a list of existing keys.
+1. Obtain a PEM formatted SSH keypair that Terraform Cloud can use to download modules during a Terraform run. You might already have an appropriate key; if not, create one on a secure workstation and distribute the public key to your VCS provider(s). Do not use or generate a key that has a passphrase. Git is running non-interactively and cannot prompt for it.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx:27]:
```suggestion
1. Obtain a PEM formatted SSH keypair that Terraform Cloud can use to download modules during a Terraform run. You might already have an appropriate key. If not, create one on a secure workstation and distribute the public key to your VCS provider(s). Do not use or generate a key that has a passphrase. Git is running non-interactively and cannot prompt for it.
```

Suggestion to remove semicolon and replace with a period and new sentence.

File: website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx [hunk 5]
```diff
@@ -19,26 +19,32 @@ To assign a key to a workspace, go to its settings and choose a previously added
...
+
+   The exact command to create a PEM formatted SSH keypair depends on your operating system. The following example command creates a `service_terraform` file with the private key, and a `service_terraform.pub` file with the public key.
```
Comment (mgarrell777) [inline website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx:29]:
```suggestion
   The exact command to create a PEM formatted SSH keypair depends on your operating system. The following example command creates a `service_terraform` file with the private key and a `service_terraform.pub` file with the public key.
```

Unnecessary comma

**Pull request merged**